packages/fluentui/react-northstar/package.json

```diff
@@ -67,6 +67,7 @@
   "scripts": {
     "build": "gulp bundle:package:no-umd && yarn build:info",
     "build:info": "gulp build:component-info",
+    "bundle-size-auditor": "bundle-size-auditor --report-path='../../../dist/bundle-size-auditor/react-northstar'",
```

Comment:
we copy reports to `'../../../dist/bundle-size-auditor/react-northstar'` here , so at the end `merge` can happen on data generated within `dist/bundle-size-auditor` folder. these data are then uploaded do storage + lightrail service